packages/components/src/input-control/input-base.tsx

```diff
@@ -64,7 +64,7 @@ function getUIFlexProps( labelPosition?: LabelPosition ) {
 	return props;
 }
 
-export function InputBase(
```

Comment:
Removed this unused export, because a consumer can erroneously import this named version instead of the default export which is properly wrapped in a `forwardRef()`.

(This happened to me while working on this PR and it wasted a couple minutes of my life 😇)

Reply:
Nit, but, what is the current convention for components - should we provide both (named, default) or just one - and if so, it seems `default` is preferred?

Another approach would be to connect before while assigning to a `const` and then export as needed.

Reply:
Good question. We do indeed have [some conventions](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/CONTRIBUTING.md#considerations-for-the-docgen) that are actually for accommodating our current docgen, more than anything else.

Doesn't matter that much for internal components like `InputBase`, which aren't fed into the docgen.